Tests/CommonMarkTests/NodeHashableTests.swift

```diff
@@ -0,0 +1,64 @@
+import XCTest
+import CommonMark
+
+final class NodeHashableTests: XCTestCase {
+    func testNodesEquality() {
+        let firstNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
+        let secondNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
+
+        XCTAssertNotEqual(firstNode, secondNode)
+        XCTAssertEqual(firstNode, firstNode)
+        XCTAssertEqual(secondNode, secondNode)
+    }
+
+    func testNodesHashable() {
+        let firstNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
+        let secondNode = Paragraph(text: #"""
+        All human beings are born free and equal in dignity and rights.
+        They are endowed with reason and conscience
+        and should act towards one another in a spirit of brotherhood.
+        """#, replacingNewLinesWithBreaks: true)
+
+        var map = [Node: String]()
+        map[firstNode] = "first"
+        map[secondNode] = "second"
+
+        XCTAssertEqual(map[firstNode], "first")
+        XCTAssertEqual(map[secondNode], "second")
+    }
+
+    func testUpdatingValue() {
```

Comment:
Although the meaning is more precise within the context of the test case, I like for test case names to be more self-descriptive. I find XCTest's default test output is hard to parse (both for humans and computers), so I tend to design test names to make it easy to tell what failed.

Here's one suggestion: 

```suggestion
    func testHashValueConsistentAfterModification() {
```

Reply:
Hahaha, this feels as if you could see the troubles I had in finding a good name for this test case. 👍 That's a perfect name for it, I will take it.